x-pack/plugins/ml/public/application/components/field_stats_flyout/field_stats_flyout_provider.tsx

```diff
@@ -6,10 +6,24 @@
  */
 
 import React, { useCallback, useState } from 'react';
-import { FieldStatsFlyout } from './field_stats_flyout';
+import { type DataView } from '@kbn/data-plugin/common';
+import { type FieldStatsServices } from '@kbn/unified-field-list-plugin/public';
+import { TimeRangeMs } from './field_stats_content';
 import { MLJobWizardFieldStatsFlyoutContext } from './use_field_stats_flytout_context';
+import { FieldStatsFlyout } from './field_stats_flyout';
+
+export const FieldStatsFlyoutProvider = ({
+  dataView,
+  fieldStatsServices,
+  timeRangeMs,
+  children,
+}: {
+  children: React.ReactElement;
+  fieldStatsServices: FieldStatsServices;
+  timeRangeMs?: TimeRangeMs;
 
```

Comment:
Nit: This line break can be removed and the prop definition and destructured function arguments could be in the same order.

Reply:
Updated here [`15484a2` (#149879)](https://github.com/elastic/kibana/pull/149879/commits/15484a2d81ddb770cc9ffa3def370b3d46f0a2b7)